wheel/python/clvm_rs/clvm_storage.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional, Tuple, _SpecialForm, cast
+
+# we support py3.7 which doesn't yet have typing.Protocol
+
+try:
+    from typing import Protocol
+except ImportError:
+    Protocol = cast(_SpecialForm, object)
+
+
+class CLVMStorage(Protocol):
+    atom: Optional[bytes]
+
+    @property
+    def pair(self) -> Optional[Tuple["CLVMStorage", "CLVMStorage"]]:
```

Comment:
I'm on the fence about this, since "the future" on this was supposed to happen in py310, but it was delayed once, so I'm not 100% convinced it's going to happen. In any case, both syntaxes will likely have to be supported for some time, so we can transition once py311 is deprecated (which is a long time from now).

Reply:
Possibly historically, but it seems to find type bugs just fine. It supports recursive types now too, which is a huge relief, since there's not really a way to define clvm types without it.